client/my-sites/domains/domain-management/dataviews/style.scss

```diff
@@ -0,0 +1,234 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/dataviews/build-style/style.css";
+
+body.is-section-domains.is-bulk-all-domains-page {
+	.layout__content {
+		padding-left: calc(var(--sidebar-width-max));
+		padding-right: 16px;
+
+
+		main.dataviews {
+			margin: 0;
+			padding-bottom: 0;
+			max-width: none;
+			height: calc(100vh - var(--masterbar-height) - var(--content-padding-top) - var(--content-padding-bottom));
+			display: flex;
+			flex-direction: column;
+			flex-wrap: nowrap;
+			overflow-y: auto;
+
+			.navigation-header {
+				padding-top: 24px;
+
+				.formatted-header {
+					max-height: 41px;
+				}
+
+				.navigation-header__main {
+					align-items: center;
+				}
+
+				.formatted-header__title {
+					color: var( --studio-gray-80, #2c3338 );
+					font-size: 1.5rem;
+					font-style: normal;
+					font-weight: 500;
+					line-height: 1.2;
+				}
+
+				.domain-header__buttons .button {
+					line-height: 22px;
+					white-space: nowrap;
+					margin-left: 0;
+				}
+
+				.domain-header__buttons-mobile {
+					white-space: nowrap;
+				}
+			}
+
+			.domains-dataviews {
+				flex-grow: 1;
+				margin-top: 0;
+				overflow: hidden;
+				padding-bottom: 0;
+
+				.dataviews-wrapper {
+					margin: auto;
+
+					.dataviews-view-list {
+						.dataviews-view-list__media-wrapper {
```

Comment:
We should remove all this CSS targeting DataViews internal component.

Reply:
Fixed here: https://github.com/Automattic/wp-calypso/pull/97677/commits/3b267fdac1be6f7a1f9e87eb84260cf3eaeaa7ce